ocs_ci/helpers/sanity_helpers.py

```diff
@@ -183,3 +187,4 @@ def __init__(self):
         self.pod_objs = list()
         self.obc_objs = list()
         self.ceph_cluster = CephClusterExternal()
+        self.ceph_cluster_external = CephClusterExternal()
```

Comment:
Why do we need to create this duplicate? We already have `ceph_cluster` from above line.

Reply:
@pintojoy Would you be able to answer this?

Reply:
@am-agrawa I have taken care of it in latest changes